src/components/Search.js

```diff
@@ -88,6 +88,8 @@ export const Search = () => {
     articleHistory.clear()
     if (!consentGranted && isOnline) {
       showConsentPopup()
+    } else if (!isOnline) {
+      closeConsentPopup()
     } else {
       setNavigation(0)
     }
```

Comment:
actually just

```js
} else {
  closeConsentPopup();
  setNavigation(0);
}
```

should work, right?

Reply:
I think you're right!